io/eolearn/tests/test_raster_io.py

```diff
@@ -360,3 +360,33 @@ def test_time_dependent_feature_with_timestamps(test_eopatch):
     new_eopatch = import_task(test_eopatch, filename=filename)
 
     assert_array_equal(new_eopatch[feature], test_eopatch[feature])
+
+
+@pytest.mark.parametrize(
+    "no_data_value, data_type", [(None, np.uint16), (np.nan, float), (0, int), (None, float), (1, np.byte)]
+)
+def test_export_import_sequence(no_data_value, data_type):
+    eopatch = EOPatch()
+    eopatch.bbox = BBox((0, 0, 1, 1), crs=CRS.WGS84)
+    feature = (FeatureType.DATA_TIMELESS, "DATA")
+
+    np_arr = np.zeros((10, 10, 1), dtype=data_type)
+    np_arr[:5, :5, :] = 1
+    eopatch[feature] = np_arr
+
+    with tempfile.TemporaryDirectory() as tmp_dir_name:
+        filename = "test_seq.tiff"
+        file_path = os.path.join(tmp_dir_name, filename)
+        export_task = ExportToTiffTask(
+            feature=feature, folder=tmp_dir_name, band_indices=[0], no_data_value=no_data_value
+        )
+        export_task.execute(eopatch=eopatch, filename=filename)
+
+        with rasterio.open(file_path) as src:
+            tif_array = src.read(masked=True)
+            assert np.sum(tif_array.mask) == np.sum(np_arr == no_data_value)
```

Comment:
could we just compare `tif_array.mask` with `np_arr == no_data_value`?

Reply:
if you add a comment describing the difference in dimensions then i think the `squeeze` approach is fine. Another option is to use `tif_array[0]` and `np_arr[..., 0]` (or whichever way it has to be), but perhaps squeeze is better here